spec/system/prior_authority/view_applications_spec.rb

```diff
@@ -7,8 +7,6 @@
     travel_to arbitrary_fixed_date
     visit provider_saml_omniauth_callback_path
     application
-    visit prior_authority_applications_path
```

Comment:
How come we're un-DRYing this? If there's one test that needs _not_ to be on this URL, could we use a context/describe block for all the others that contains this stuff in a `before` so we don't need to copy-paste so much?